### PR TITLE
f/wpc-576: highlight slug field and disable publish button when no slug

### PR DESCRIPTION
**Pull request by camilleory**
The slug field gets a red highlight and the publish button is disabled when the article doesn't have a slug
The test has been adapted.

image

### CODE REVIEW

File: packages/editor/src/client/api/index.ts [hunk 1]
```diff
@@ -1731,6 +1737,7 @@ export type User = {
   properties: Array<Properties>;
   roles: Array<UserRole>;
   subscription?: Maybe<UserSubscription>;
+  paymentProviderCustomers: Array<PaymentProviderCustomer>;
```
Comment (penk85) [inline packages/editor/src/client/api/index.ts:1740]: What's this from?

Comment (camilleory) [reply packages/editor/src/client/api/index.ts:1740]: I think it comes from Nico's branch (the one he updated the payment type). Not sure why it comes in my code. @roosnic1 is it problematic ?

**Review by lakeside763 — APPROVED** [review]

File: packages/editor/src/client/api/index.ts (hunk 1 above)
Comment (roosnic1) [reply packages/editor/src/client/api/index.ts:1740]: @penk85 and @camilleory  Yeah i forgot to commit that file. Is fixed here https://github.com/wepublish/wepublish/pull/420 please review and once merged this should be gone.

**Review by penk85 — APPROVED** [review]
Looks great

**Review by roosnic1 — CHANGES_REQUESTED** [review]
I recommend to wait with this PR and reuse the component from https://github.com/wepublish/wepublish/pull/418

**Pull request merged**